delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/directives/DeltaDirectives.scala

```diff
@@ -0,0 +1,419 @@
+package ch.epfl.bluebrain.nexus.delta.routes.directives
+
+import java.util.UUID
+
+import akka.http.scaladsl.marshalling.sse.EventStreamMarshalling._
+import akka.http.scaladsl.model.MediaTypes.`application/json`
+import akka.http.scaladsl.model.StatusCodes.OK
+import akka.http.scaladsl.model._
+import akka.http.scaladsl.model.headers.`Last-Event-ID`
+import akka.http.scaladsl.model.sse.ServerSentEvent
+import akka.http.scaladsl.server.ContentNegotiator.Alternative
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server._
+import akka.persistence.query.{NoOffset, Offset, Sequence, TimeBasedUUID}
+import akka.stream.scaladsl.Source
+import ch.epfl.bluebrain.nexus.delta.rdf.RdfError
+import ch.epfl.bluebrain.nexus.delta.rdf.graph.{Dot, NTriples}
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.context.{ContextValue, RemoteContextResolution}
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.{JsonLd, JsonLdEncoder}
+import ch.epfl.bluebrain.nexus.delta.rdf.utils.JsonKeyOrdering
+import ch.epfl.bluebrain.nexus.delta.routes.marshalling.RdfMediaTypes._
+import ch.epfl.bluebrain.nexus.delta.routes.marshalling.{HttpResponseFields, JsonLdFormat, RdfMarshalling}
+import ch.epfl.bluebrain.nexus.delta.sdk.error.ServiceError
+import ch.epfl.bluebrain.nexus.delta.sdk.model.search.SearchResults
+import ch.epfl.bluebrain.nexus.delta.sdk.model.search.SearchResults._
+import ch.epfl.bluebrain.nexus.delta.sdk.model.{Envelope, Event}
+import ch.epfl.bluebrain.nexus.delta.syntax._
+import fs2.Stream
+import monix.bio.{IO, Task, UIO}
+import monix.execution.Scheduler
+import streamz.converter._
+
+import scala.util.Try
+
+object DeltaDirectives extends UriDirectives with RdfMarshalling {
```

Comment:
Some documentation in the second part could help better understand and separate more the different blocks

Reply:
For now I've changed the method names so it should be quite self explanatory looking at it and at the parameters it requires.

Reply:
Ok, it was also because at least when reviewing due to the github layout, it is not so easy to see where starts and ends the main parts ToResponse/DiscardResponse/...